docs/content/user-guide/en/transport/azure-service-bus.md

```diff
@@ -109,3 +109,30 @@ c.UseAzureServiceBus(asb =>
         };
 });
 ```
+
+#### Custom Consumers
+
+You can customize the setup for customer consumers in Azure by configuring extra service options per group. This includes integrating with extra topic or adjusting other relevant settings.
+
+```C#
+c.UseAzureServiceBus(asb =>
+{
+    asb.ConnectionString = ...
+           asb.ConfigureCustomGroupConsumer("test", cfg =>
+        {
+            cfg.UseTopic("entity-created");
+            cfg.UseConnectionString("external connection string");
```

Comment:
Do you think we should also remove this here since all consumers should be in the same namespace defined in the default config?